### PR TITLE
[MUI-418] MNTReward table component

### PR DESCRIPTION
**Pull request by p1geon**
# Description

Created MNTReward table component

## Jira ticket(optional)

[MUI-418](https://minterestfinance.atlassian.net/browse/MUI-418)

## Type of change

Please pick options that are relevant.

- [ ] Bug fix OR refactoring (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature or refactoring that would cause existing functionality to not work as expected)

PLEASE DON'T CHECK ASSET NAME COMPONENT

### CODE REVIEW

File: src/components/Table/AssetName/AssetName.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+})<{ isHovered?: boolean }>(({ isHovered }) => ({
+  marginLeft: 8,
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: isHovered ? 'space-between' : 'center',
+}));
+
+const Amount = styled(Typography)({
+  marginLeft: 4,
+  color: '#6D7692',
```
Comment (mr-teddy-bear) [inline src/components/Table/AssetName/AssetName.tsx:40]: color from theme?

File: src/components/Table/AssetName/AssetName.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  fontSize: 14,
+  lineHeight: 1.5,
+});
+
+const UnderlyingWrapper = styled('div')({
+  display: 'flex',
+  '&:hover': {
+    cursor: 'pointer',
+    '& .MuiTypography-root': {
+      color: '#061953',
```
Comment (mr-teddy-bear) [inline src/components/Table/AssetName/AssetName.tsx:55]: color from theme?

File: src/components/Table/AssetName/AssetName.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  flexDirection: 'column',
+  justifyContent: isHovered ? 'space-between' : 'center',
+}));
+
+const Amount = styled(Typography)({
+  marginLeft: 4,
+  color: '#6D7692',
+});
+
+const Title = styled(Typography)({
```
Comment (mr-teddy-bear) [inline src/components/Table/AssetName/AssetName.tsx:43]: Do we need use font-family and font-weight if we use Typography?

File: src/components/Table/AssetName/AssetName.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  '&:hover': {
+    cursor: 'pointer',
+    '& .MuiTypography-root': {
+      color: '#061953',
+    },
+  },
+});
+
+const TooltipTitle = styled(Typography)({
+  color: '#6D7692',
```
Comment (mr-teddy-bear) [inline src/components/Table/AssetName/AssetName.tsx:61]: color from theme?

File: src/components/Table/MNTReward/MNTReward.tsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  width: 12,
+  height: 12,
+});
+
+const CustomButton = styled(ButtonUnstyled)({
+  width: '100%',
+  height: 20,
+  borderRadius: 4,
+  border: '1px solid rgba(12, 45, 156, 0.08)',
+  backgroundColor: '#FCFCFC',
```
Comment (mr-teddy-bear) [inline src/components/Table/MNTReward/MNTReward.tsx:34]: color from theme?

File: src/components/Table/MNTReward/MNTReward.tsx [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+  display: 'flex',
+  alignItems: 'center',
+  paddingLeft: 3,
+  paddingRight: 0,
+  '&:hover': {
+    backgroundColor: '#F0F1F2',
+    cursor: 'pointer',
+    '& .rewardValue': {
+      color: '#061953',
```
Comment (mr-teddy-bear) [inline src/components/Table/MNTReward/MNTReward.tsx:43]: color from theme?

File: src/components/Table/AssetName/AssetName.tsx (hunk 1 above)
Comment (p1geon) [reply src/components/Table/AssetName/AssetName.tsx]: fixed

File: src/components/Table/AssetName/AssetName.tsx (hunk 2 above)
Comment (p1geon) [reply src/components/Table/AssetName/AssetName.tsx]: fixed

File: src/components/Table/AssetName/AssetName.tsx (hunk 3 above)
Comment (p1geon) [reply src/components/Table/AssetName/AssetName.tsx]: fixed

File: src/components/Table/MNTReward/MNTReward.tsx (hunk 6 above)
Comment (p1geon) [reply src/components/Table/MNTReward/MNTReward.tsx]: fixed

**Review by marinakulsha — APPROVED** [review]

**Review by mr-teddy-bear — APPROVED** [review]

**Review by gavrilin-remote — APPROVED** [review]
ok for now, but I will think about how to optimize our palette

**Review by mr-teddy-bear — APPROVED** [review]

**Pull request merged**